src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/specify-when-alerts-create-incidents.mdx

```diff
@@ -125,63 +135,69 @@ Repeat these steps for each policy as needed.
+  Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Alert conditions (policies) > (select a policy)**. The selected policy page shows how alerts rolls up issues for alert notifications and UI details. The default option is **One issue per policy**. To choose a different option for this policy, click **Issue creation preference**.
 </figcaption>
 
 ## One issue per policy (default) [#preference-policy]
 
+Choose the **One issue per policy** preference if your policy and its conditions have a focused scope, like monitoring one entity.
+
+If additional conditions open incidents after the issue opens, the incidents are added to the same issue. You will only see one issue on your **Issues & activity** page for the selected policy. Learn more about [your issue creation preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#issue-creation).
+
 <img
-  title="A diagram showing how the By policy issue preference works"
-  alt="A diagram showing how the By policy issue preference works"
+  width="40%;"
+  class="center"
+  title="A diagram showing how the **One issue per policy** option works"
+  alt="A diagram showing how the **One issue per policy** option works"
   src={alertsByPolicy}
 />
 
 <figcaption>
-  When you select the default (**One issue per policy**) as your issue preference, it will group all condition incidents into the same issue.
+  When you select the default option **One issue per policy** as your issue preference, it will group all condition incidents into the same issue.
 </figcaption>
 
-If additional conditions open incidents after the issue opens, the incidents are added to the same issue. Only one issue appears on your **Issues & activity** page for the selected policy.
+## One issue per condition [#preference-condition]
 
-Choose the **One issue per policy** preference if your policy and its conditions have a focused scope (like monitoring one entity). Learn more about [issue preference best practices](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#incident-practices).
+Choose the **One issue per condition** option when you want an individual issue record to focus on a specific condition. If the policy has multiple conditions, you will see separate issue records for each condition on your **Issues & activity** page. Learn more about [your issue creation preferences](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/#issue-creation).
 
-## One issue per condition [#preference-condition]
+* If a target breaches a condition for the alert policy, an incident opens and it's created an issue record.
+* If the same target breaches another condition for the alert policy, another incident opens and it's created another issue record.
```

Comment:
"If the same signal breaches another condition's threshold, another incident opens and another issue record is created."

Again, thrown by the "for the alert policy" and "it's"